pkg/models/settings.go

```diff
@@ -235,6 +246,10 @@ func LoadSettings(ctx context.Context, config backend.DataSourceInstanceSettings
 		if infJson.TimeoutInSeconds > 0 {
 			settings.TimeoutInSeconds = infJson.TimeoutInSeconds
 		}
+		settings.UnsecuredQueryHandling = infJson.UnsecuredQueryHandling
+		if settings.UnsecuredQueryHandling == "" {
+			settings.UnsecuredQueryHandling = UnsecuredQueryHandlingWarn
+		}
```

Comment:
nitpick.... would it make sense to validate that it has a valid value, and not let's say `gabor`? 😄